app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id', dependent: :destroy
+  has_many :comments, foreign_key: 'author_id', dependent: :destroy
+  has_many :likes, foreign_key: 'author_id', dependent: :destroy
+
+  # def last_3_posts
```

Comment:
As part of the requirement, The user model should have a method that returns the 3 most recent posts. Please create a method in `user.rb` that returns the last 3 posts. 👍🏻

Reply:
Thanks 👍🏻 

I completely forgot to do that.